registry/recipes/packages/py-prompthandler/src/faktion/prompthandler/in_memory.py

```
@@ -0,0 +1,93 @@
+        for name, value in module_attrs.items():
+            if name.startswith("_"):
+                continue
+            if not name.isupper():
+                continue
+            if not isinstance(value, str):
+                continue
+
+            prompts[name] = value
+
+        if not prompts:
+            raise ValueError("No prompts found in module", prompt_module.__package__, list(module_attrs.keys()))
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)